### PR TITLE
Listing Code owners

### LINKED ISSUE #2720: Consider listing code owners for this repository
When submitting a PR updating code you're not familiar with, it's not always easy to determine who's the right person it should be assigned to for review (e.g when modifying the modular middleware, chances are fairly high you'll want JTK to take a look :smile:).

To fix that, I suggest establishing a list of code owners who will be automatically pinged when code they own is updated by a PR: https://blog.github.com/2017-07-06-introducing-code-owners/

Needless to say, I'm applying to be one of the code owners of the OpenID module/core components :sweat_smile:

### PR DESCRIPTION
**Pull request by agriffard**
Fixes #2720

Please comment if you see some other possible owners/features.

### CODE REVIEW

**Comment by hishamco** [comment]
You could add a `Translation`  feature  to me too ;) more specifically if there's PS scripts needed, but if the case is Crowdin it's fine that you own it

**Comment by hishamco** [comment]
Also you can add `UI` feature to me @Skrypt

File: src/docs/resources/owners/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+| Owner | Area |
+|--- | --- |
+| @jtkech | Infrastructure, Build, Distributed Services |
+| @sfmskywalker | Forms, Workflows |
+| @Jetski5822 | GraphQL |
+| @carlwoodhouse | GraphQL |
+| @PinpointTownes | OpenID |
+| @deanmarcussen | Azure, Media, Shortcodes, Sitemaps, Taxonomies |
+| @agriffard | Docs, Crowdin Translations, Try, Website |
```
Comment (hishamco) [inline src/docs/resources/owners/README.md:13]: Try is not meaningful, possible is try.OrachardCore

**Comment by hishamco** [comment]
matiasmolleja => Media
hishamco => UI, Email

**Comment by kevinchalet** [comment]
It would be nice to also have a CODEOWNERS file in the `.github` so that owners are automatically notified to changes made to code they "own".

File: src/docs/resources/owners/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The below table lists the different code/features areas and their owners:
+
+| Owner | Area |
+|--- | --- |
+| @jtkech | Infrastructure, Build, Distributed Services |
+| @sfmskywalker | Forms, Workflows |
+| @Jetski5822 | GraphQL |
+| @carlwoodhouse | GraphQL |
+| @PinpointTownes | OpenID |
```
Comment (kevinchalet) [inline src/docs/resources/owners/README.md:11]: `@PinpointTownes` -> `@kevinchalet`

Thanks 😃

**Comment by agriffard** [comment]
CODEOWNERS file added.

**Pull request merged**